### PR TITLE
fix: transaction info styling

### LINKED ISSUE #1523: Fix styles for the transaction info: hash, created, exectuted etc.
## Bug description
Apply the styles of tx info according to the designs.https://www.figma.com/file/ptTs6lDBeUuLNySroJ5PiF/Web-Master-File?node-id=265%3A9731&t=dk9zw6NEGRIhAhHG-1
Currently the bold black font is taking to much attention comparing to the other elements. 

## Environment
 - Browser: Chrome
 - Wallet: MetaMask
 - Chain: Ethereum mainnet
Screenshot 2023-01-11 at 13 22 18

## Steps to reproduce
 1. Go to

### PR DESCRIPTION
**Pull request by abheektripathy**
## What it solves

Resolves #1523 

## How this PR fixes it
changed css, used `text.secondary` as color property.

## Screenshots
image

### CODE REVIEW

File: src/components/transactions/TxDetails/Summary/TxDataRow/index.tsx [hunk 1]
```diff
@@ -15,7 +15,9 @@ export const TxDataRow = ({ title, children }: TxDataRowProps): ReactElement | n
   if (children == undefined) return null
   return (
     <div className={css.gridRow}>
-      <div className={css.title}>{title}</div>
```
Comment (katspaugh) [inline src/components/transactions/TxDetails/Summary/TxDataRow/index.tsx:18]: You could keep the div and just use `color: var(--color-text-secondary);` in the CSS file.

Comment (abheektripathy) [reply src/components/transactions/TxDetails/Summary/TxDataRow/index.tsx:18]: on it.

File: src/components/transactions/TxDetails/Summary/TxDataRow/styles.module.css [hunk 2]
```diff
@@ -11,6 +11,12 @@
   word-break: break-all;
 }
 
+.detailstitle {
+  font-weight: 400;
+  word-break: break-all;
+  color: var(--color-text-secondary);
```
Comment (iamacook) [inline src/components/transactions/TxDetails/Summary/TxDataRow/styles.module.css:17]: Adding a new class is not necessary. The `color` can be appended to `title` as it is only used in `TxDataRow`.

**Review by iamacook — APPROVED** [review]

File: src/components/transactions/TxDetails/Summary/TxDataRow/styles.module.css [hunk 3]
```diff
@@ -9,6 +9,7 @@
 .title {
   font-weight: 700;
```
Comment (katspaugh) [inline src/components/transactions/TxDetails/Summary/TxDataRow/styles.module.css:10]: Can you also change this to 400?

Comment (abheektripathy) [reply src/components/transactions/TxDetails/Summary/TxDataRow/styles.module.css:10]: sure.

**Review by katspaugh — APPROVED** [review]
Looks great, thank you @abheektripathy!

**Pull request merged**